man/digest.Rd

```diff
@@ -1,9 +1,9 @@
 \name{digest}
 \alias{digest}
-\title{Create hash function digests for arbitrary R objects}
+\title{Create hash function digests for arbitrary R objects or files}
```

Comment:
I am on the fence here as titles should be short and succinct.  I will probably merge the PR but may remove these two words later -- will ponder.